lib/iBeacon/README.md

```diff
@@ -1,15 +1,13 @@
-#
-# DEVELOPER IS RESPONSIBLE FOR OBTAINING THE NECESSARY LICENSES FROM APPLE.
-#
-# iBeacon(TM) is a trademark of Apple Inc. and use of this code must comply with
-# their licence.
-#
-
-iBeacon Library
-======================================
+iBeacon Library (SEE LICENSE NOTICE)
+====================================
 
 MicroPython library for creating and parsing iBeacon frames.
 
+**DEVELOPER IS RESPONSIBLE FOR OBTAINING THE NECESSARY LICENSES FROM APPLE.
```

Comment:
@diescalo Will `PATTERN_BOLD_TEXT` match this? Or would it be better if I put `**` around each line in the paragraph?

Reply:
Yes, it will, just checked.